workspace/viewer/PdfJsViewer.ts

```diff
@@ -13,11 +13,12 @@ interface PDFPageViewStatic {
   new(options: any) : PDFPageViewStatic;
   destroy(): void;
 
-  setPdfPage(pdfPage: PDF.PDFPageProxy): void;
+  setPdfPage(pdfPage:  PDF.PDFPageProxy): void;
```

Comment:
why this space?

Reply:
Done.

Reply:
I meant the space symbol after `pdfPage:`. There are two instead of one, the extra space was added by you in [this commit](https://github.com/bardsoftware/papeeria-pdfjs/pull/1/commits/5ba7e3dd2c135a3081ee1ef92d890f4ba4e3ddf7)

Reply:
Done.